manim/utils/testing/frames_comparison.py

```diff
@@ -31,13 +33,13 @@
 
 
 def frames_comparison(
-    func=None,
+    func: Callable | None = None,
     *,
     last_frame: bool = True,
-    renderer_class=CairoRenderer,
-    base_scene=Scene,
-    **custom_config,
-):
+    renderer_class: type[CairoRenderer | OpenGLRenderer] = CairoRenderer,
+    base_scene: type[Scene] = Scene,
+    **custom_config: Any,
+) -> Any:
```

Comment:
This function could use a more specific `Callable` for `func` and an overload depending on whether this is a `Callable` (return the decorated function) or `None` (return the decorator itself), but properly typing this is much more complex, so this is probably fine for now.

However, in both cases, this always returns a `Callable` anyways.
```suggestion
) -> Callable:
```